testplan/common/utils/sockets/fix/server.py

```diff
@@ -367,8 +367,21 @@ def active_connections(self):
           target)
         :rtype: ``list`` of ``tuple`` of ``str`` and ``str``
         """
+
+        # from PEP-0469 (common subset of Python 2 and 3)
+        try:
+            dict.iteritems
+        except AttributeError:
+            # Python 3
+            def itervalues(d):
+                return iter(d.values())
+        else:
+            # Python 2
+            def itervalues(d):
+                return d.itervalues()
+
         return [detail.name
-                for detail in self._conndetails_by_fd.itervalues()
+                for detail in itervalues(self._conndetails_by_fd)
```

Comment:
I think we can skip the new code addition that defines an itervalues method and we can simple change itervalues() to values() that works in both versions.
Is that ok?

Reply:
Ah yes perfect! Thank you.